README.md

```diff
@@ -67,6 +67,31 @@ There are a few categories of tests in this repo:
 - Github Actions & Nightly tests
 
 
+Code Quality Assurance
+======================
+
+This project uses rustfmt and clippy to provide a unified code style.
+When opening pull requests, it is advised to run the following commands
+before doing so:
+
+```bash
+$ cargo clippy --all-features --workspace --all-targets -- \
```

Comment:
Should we include a shell script to run this, or some such?

Reply:
Or use a config file?

Reply:
Last time I checked, this was not possible to configure with `.config/clippy.toml`, but now it apparently is, with `Cargo.toml`!